src/vs/workbench/contrib/codeEditor/browser/emptyTextEditorHint/emptyTextEditorHint.ts

```diff
@@ -395,11 +405,30 @@ class EmptyTextEditorHintContentWidget implements IContentWidget {
 		const keybindingsLookup = [ChangeLanguageAction.ID, ApplyFileSnippetAction.Id, 'welcome.showNewFileEntries'];
 		const keybindingLabels = keybindingsLookup.map((id) => this.keybindingService.lookupKeybinding(id)?.getLabel() ?? id);
 		const ariaLabel = localize('defaultHintAriaLabel', 'Execute {0} to select a language, execute {1} to fill with template, or execute {2} to open a different editor and get started. Start typing to dismiss.', ...keybindingLabels);
+		this.firstHintStatement = hintElement.querySelectorAll('a')[0];
+		const hintLanguageMappings: { [key: string]: string } = {
+			'Select a language': '0',
+			'fill with template': '1',
+			'open a different editor': '2',
+			'don\'t show': '3'
+		};
 		for (const anchor of hintElement.querySelectorAll('a')) {
 			anchor.style.cursor = 'pointer';
+			anchor.tabIndex = 0;
 			const id = keybindingsLookup.shift();
 			const title = id && this.keybindingService.lookupKeybinding(id)?.getLabel();
 			hintHandler.disposables.add(this.hoverService.setupManagedHover(getDefaultHoverDelegate('mouse'), anchor, title ?? ''));
+
+			hintHandler.disposables.add(dom.addStandardDisposableListener(anchor, 'keydown', (event) => {
+				if (event.keyCode === KeyCode.Enter && event.target.innerText in hintLanguageMappings) {
+					event.stopPropagation();
+					event.preventDefault();
+					hintHandler.callback(hintLanguageMappings[event.target.innerText], event);
```

Comment:
This won't work as expected for users with a configured display language because the display message above is localized and the keys in `hintLanguageMappings` are not. I'd also prefer not to duplicate text since we may change the wording in future. One alternative is to use the index when iterating through `hintElement.querySelectorAll('a')`:

```ts
		hintElement.querySelectorAll('a').forEach((anchor, index) => {
			// ...
			hintHandler.callback(String(index), event);
```